packages/astro-angular/src/client.ts

```diff
@@ -1,12 +1,15 @@
 import 'zone.js/dist/zone.js';
 import {
+  DestroyRef,
   EnvironmentProviders,
   Provider,
   reflectComponentType,
   ɵComponentType as ComponentType,
 } from '@angular/core';
 import { ApplicationRef, NgZone, createComponent } from '@angular/core';
+import { takeUntilDestroyed } from '@angular/core/rxjs-interop';
```

Comment:
This would be a breaking change for v15 projects, so we'll either need to change it to use a Subject/ngOnDestroy or drop support for using Angular v15 in this project.